autoload/neomake/makers/ft/d.vim

```diff
@@ -0,0 +1,50 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#d#EnabledMakers()
```

Comment:
If it makes sense to follow my comment in `s:UpdateDub`, this function could just be:

```
function! neomake#makers#ft#d#EnabledMakers()
    return ['dmd']
endfunction
```